src/Files.App/Data/Models/ColumnViewModel.cs

```diff
@@ -19,7 +19,7 @@ public bool IsHidden
 		[LiteDB.BsonIgnore]
 		public double MaxLength
 		{
-			get => UserCollapsed ? 0 : NormalMaxLength;
+			get => UserCollapsed || IsHidden ? 0 : NormalMaxLength;
```

Comment:
What does adding `IsHidden` do?

Reply:
Double check.

Reply:
UserCollapsed means user has hidden the column
IsHidden means the column is hidden b/c it's not a git folder